### ISSUE TITLE
License Meta information

### ISSUE THREAD

**Issue by ghost** [TedDriggs/serde_case#4]
Hello, the Cargo.toml mentions an MIT / Apache 2 Dual license, yet the repository only contains an MIT License file. Would you be so kind to clarify which of applies?

**Comment by TedDriggs** [comment]
I think the dual is correct, but I need to check the source code for `serde` since that's where this came from. Are you working on a project that requires one or the other?

**Comment by ghost** [comment]
No, yet I am required to keep track of the licenses we use in our stack. Such mismatches between License files and meta information require manual work, since it cannot be handled by our tooling.

**Comment by guellaeq** [comment]
Hello, one more thing, license duality defined with "/" is ambiguous. Currently, it is impossible to know if this package is licensed with an "AND" or an "OR" between the two licenses

example in the Cargo.toml file from the rust docs : https://doc.rust-lang.org/cargo/reference/manifest.html#the-license-and-license-file-fields 
```toml
license = "MIT OR Apache-2.0"
```
